functional test framework for CSI driver (#76)
openshift/csi-driver-shared-resource: test/functional/README.md

> # Functional test for OpenShift Shared Resource CSI Driver

Reviewer: ```suggestion
# QA for OpenShift Shared Resource CSI Driver
```

> This is a Behavior-driven development (or BDD) test framework leveraging behave for python. BDD is an agile software development technique that encourages collaboration between developers, QA and non-technical or business participants in a software project.Behave uses tests written in a natural language style, backed up by Python code.

Reviewer: ```suggestion
This is a Behavior-driven development (or BDD) test framework leveraging `behave` for `python`. BDD is an agile software development technique that encourages collaboration between developers, QA and non-technical or business participants in a software project.Behave uses tests written in a natural language style, backed up by Python code.
```

> We are using this to test the functionality of Openshift Shared Resource CSI Driver

Reviewer: ```suggestion
We are using this framework to achieve the QA verification of any Jira Stories development takes on for the OpenShift Shared Resource CSI Driver, as well as any regression test suites the team want to cultivate for the OpenShift Shared Resource CSI Driver.

Hence, the reader should consider these tests the 'next phase' of testing after the golang based tests in https://github.com/openshift/csi-driver-shared-resource/tree/master/test/e2e that run in OpenShift CI prow jobs as part of merging pull requests with code changes.
```

# Steps to run on local machine

- pip3  install -r /test/functional/requirements.txt

> - export KUBECONFIG = path/to/kubeconfig

Reviewer 2: There shouldn't be spaces around '='  otherwise shell expression will fail.
